src/renderers/webgl/WebGLShadowMap.js

```diff
@@ -312,8 +312,9 @@ THREE.WebGLShadowMap = function ( _renderer, _lights, _objects ) {
 
 		if ( ! customMaterial ) {
 
-			var useMorphing = geometry.morphTargets !== undefined &&
-					geometry.morphTargets.length > 0 && material.morphTargets;
+			var useMorphing = ( ( geometry.morphTargets !== undefined && geometry.morphTargets.length > 0 ) || 
+				( geometry.morphAttributes.position !== undefined && geometry.morphAttributes.position.length > 0 ) ) 
+				&& material.morphTargets;
```

Comment:
Right. Since `material.morphTargets` is checked in both case I wrote this new pattern